compliance_checker/cfutil.py

```diff
@@ -53,15 +53,25 @@
 }
 
 
-def get_unitless_standard_names(xml_tree, units):
+def get_dimensionless_standard_names(xml_tree, standard_name):
```

Comment:
i'm confused as to why you changed all these to dimensionless. I thought it was just the units -> standard_name arg that needed fixing

Reply:
The terminology "unitless" doesn't appear in the CF manual.  Furthermore what were "unitless" variables could sometimes have units of a constant, i.e. '1e-3' or '1', so it wasn't truly unitless in that sense either, but these checks do refer to "dimensionless" variables, as noted in http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#units

Reply:
@mhidas, thanks for the feedback, I think your points are pretty spot on here, so I'll be implementing your suggestions.